File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.js
function solution(numbers) {
    // const arr = Array(10).fill().map((_,idx) => {return idx+1});
    const arr = Array.from( {length : 10}, (element, idx) => {return idx;});
    let answer = 0;
    arr.forEach((element,idx) => {
        numbers.includes(element) ? answer += 0 : answer += element; 
    });
    
    
    
    return answer;
}

File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.java
class Solution {
    public int solution(int[] numbers) {
        int answer = 0;
        int[] numList = new int[10];
        for (int i = 0; i < 10; i++) {
            numList[i] = 0;
        }
        
        for (int i = 0; i < numbers.length; i++) {
            numList[numbers[i]] += 1;
        }
        
        for (int i = 0; i < numList.length; i++) {
            if (numList[i] == 0) {
                answer += i;
            }
        }
        
        return answer;
    }
}